puppet-jens.spec

```
@@ -47,6 +47,9 @@ mkdir -m 750 -p %{buildroot}/var/www/jens
 %{__install} -D -p -m 755 wsgi/* %{buildroot}/var/www/jens
 mkdir -p %{buildroot}%{_tmpfilesdir}
 install -m 0644 jens-tmpfiles.conf %{buildroot}%{_tmpfilesdir}/%{name}.conf
+mkdir -p $RPM_BUILD_ROOT%{_unitdir}
```

```suggestion
mkdir -p %{buildroot}%{_unitdir}
```
Be consistent with use of `%{buildroot}` and `$RPM_BUILD_ROOT` , the former is preferred.

---

https://gitlab.cern.ch/ai-config-team/jens-build/-/pipelines/2046899